randovania/cli/commands/export_db_videos.py

```diff
@@ -99,21 +99,26 @@
 """
 
 HTML_VIDEO_FORMAT = """
-<p><i> {} </i></p>
-<iframe
-width="728"
-height="410"
-src="https://www.youtube.com/embed/{}?start={}&autoplay=1"
-srcdoc="<style>*{{padding:0;margin:0;overflow:hidden}}html,body{{height:100%}}
-img,span{{position:absolute;width:100%;top:0;bottom:0;margin:auto}}
-span{{height:1.5em;text-align:center;font:48px/1.5 sans-serif;color:white;text-shadow:0 0 0.5em black}}
-</style><a href=https://www.youtube.com/embed/{}?start={}&autoplay=1>
-<img src=https://img.youtube.com/vi/{}/hqdefault.jpg alt='vid'><span>▶️</span></a>"
-frameborder="0"
-allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-allowfullscreen
-title="vid"
-></iframe>
+        <p><i> {} </i></p>
+        <iframe
+        width="560"
```

Comment:
I can't find a reason why this would be changed. Looking at the db output of stable and main, I prefer the larger HTML frames (720x410)

Everything else doesn't seem to matter

Reply:
do you want me to just revert width/height, or the whole html snippet?

Reply:
the whole file please